lms/djangoapps/certificates/models.py

```diff
@@ -504,6 +504,12 @@ class CertificateInvalidation(TimeStampedModel):
     notes = models.TextField(default=None, null=True)
     active = models.BooleanField(default=True)
 
+    # This is necessary because CMS does not install the certificates app, but it
+    # imports this models code. Simple History will attempt to connect to the installed
+    # model in the certificates app, which will fail.
+    if 'certificates' in apps.app_configs:
+        history = HistoricalRecords()
+
```

Comment:
Curious about this bit. I see that we do this above for `GeneratedCertificate` (and a few other models in edx-platform). 

I see that `GeneratedCertificate` appears to be used in some `common` code which I assume is why we need to have this conditional above.

I only see the `CertificateInvalidation` model imported in the LMS (imported in `support.py` at `/lms/djangoapps/certificates/views`). Do we need this here? Or, am I just missing something? (which I very well may be :-)).

Reply:
You are right! Removing the condition. ✅

Reply:
Aaaaaand actually we think removing this broke the tests! Putting it back ...